index.html

```diff
@@ -58,6 +59,216 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio2.png" class="project_img" alt="project_image">
+      <h2 class="project_title">Tonic</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio3.png" class="project_img" alt="project_image">
+      <h2 class="project_title">Multi-Post Stories</h2>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+  </section>
+
+  <section class="aboutme_container">
+    <div class="aboutme_class">
+      <h2 class="identity_text">About Myself</h2>
```

Comment:
- [ ] Please Fix what was asked from the previous review. kindly stick to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
- [ ] Original Design:

![zb](https://user-images.githubusercontent.com/63972670/159113541-c8a4f1c2-1150-49fe-9245-edb34837a0af.JPG)

- [ ] From your design:

![WhatsApp Image 2022-03-19 at 9 14 38 AM](https://user-images.githubusercontent.com/63972670/159113611-8eae0ba7-35a1-4297-86c2-123320123cf3.jpeg)